server/src/internalClusterTest/java/org/elasticsearch/health/UpdateHealthInfoCacheIT.java

```diff
@@ -40,14 +42,7 @@ public void testNodesReportingHealth() throws Exception {
             String[] nodeIds = getNodes(internalCluster).keySet().toArray(new String[0]);
             DiscoveryNode healthNode = waitAndGetHealthNode(internalCluster);
             assertThat(healthNode, notNullValue());
-            assertBusy(() -> {
-                Map<String, DiskHealthInfo> healthInfoCache = internalCluster.getInstance(HealthInfoCache.class, healthNode.getName())
-                    .getDiskHealthInfo();
-                assertThat(healthInfoCache.size(), equalTo(nodeIds.length));
-                for (String nodeId : nodeIds) {
-                    assertThat(healthInfoCache.get(nodeId), equalTo(GREEN));
-                }
-            });
+            assertBusy(() -> { assertResultsCanBeFetched(internalCluster, healthNode, List.of(nodeIds), null); });
```

Comment:
Great I like where this is going :). I think the curly braces aren't necessary, would you mind removing them here and in other places?

Reply:
Done.

Reply:
Thanks!